### ISSUE TITLE
PATCH  /{realm}/localization/{locale} disappeared from API

### ISSUE THREAD

**Issue by senyahnoj** [keycloak/keycloak#12253]
### Describe the bug

In Keycloak 15 you are able to make a PATCH request to the API in order to update realm localization strings

This is documented in:

https://www.keycloak.org/docs-api/15.0/rest-api/index.html

Upgraded to Keycloak 16 and calls to this API endpoint are returning with "405 method not allowed"

Looking at the REST API documentation at

https://www.keycloak.org/docs-api/16.0/rest-api/index.html

And the entry for **PATCH /{realm}/localization/{locale}** is no longer mentioned.

When I look at the changelog for v16 https://www.keycloak.org/2021/12/keycloak-1600-released there is no mention of this modification to the REST API, therefore can I assume this is a bug?

### Version

16.1.1

### Expected behavior

PATCH  /{realm}/localization/{locale} REST API call should allow me to send up a patch of localization strings for the given locale and not throw an error

### Actual behavior

PATCH  /{realm}/localization/{locale} REST API throws a "405 Method Not Allowed" response

### Anything else?

Screenshots showing REST API docs 15 and 16

![REST-API-15](https://user-images.githubusercontent.com/5585851/171007708-dc7457af-caf9-4e1d-a404-197229ab92f3.png)
![REST-API-16](https://user-images.githubusercontent.com/5585851/171007716-59abc3b7-acc7-455c-ba07-cc8c73f4ed7f.png)

**Comment by jbman** [comment]
The same functionality is still available with **POST /auth/admin/realms/{realm}/localization/{locale}**

It update localization texts for a given realm and locale.
This method will not delete any text. It updates texts, which are already stored or create new ones if the key does not exist yet.

I assume the PATCH has been removed without writing a changelog entry and POST must be used as of version 16.

Note: Last time I checked the new admin console v2 was missing the option to import a JSON with localization entries
